hooks/push

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "Push is taken care of by buildx, during the build phase"
```

Comment:
do we need this file? maybe without this file it will push automatically? (and not use `--push`)

Reply:
I took this from other repos. Actually without this 
, it did not work, only overwriting the push helped. Buildx during hooks behaves a slightly differently

Reply:
sounds good then, thanks